content/blog/newsletter_2022-02-05.de.md

```diff
@@ -0,0 +1,235 @@
+---
+title: XMPP Newsletter Dezember 2021 und Januar 2022
+author: Anoxinon e.V. 
+date: 2022-02-05
+categories: ['Newsletter']
+---
+
+*Info: [Anoxinon e.V.](https://anoxinon.de/) publiziert die deutsche Übersetzung des unter der CC by-sa 4.0 Lizenz stehenden XMPP Newsletters für die XSF Foundation. Den [Originalartikel findest Du im Blog der XSF](https://xmpp.org/2022/02/the-xmpp-newsletter-december-2021-january-2022/). Übersetzung und Korrektur der dt. Version von Anoxinon e.V..*
+
+Willkommen zum XMPP-Newsletter für den Dezember 2021 und den Jaunar 2022!
+
+Wir hoffen, dass du bisher einen guten Übergang in das neue Jahr hattest und dich auf die neue Ausgabe freust! Wir denken, dass diese Ausgabe über die Neujahrsfeiertage einen gewissen Umfang angenommen hat.
+
+Viele Projekte und deren Bemühungen in der XMPP-Gemeinschaft sind das Ergebnis von Freiwilligenarbeit. Wenn du mit den von dir verwendeten Diensten und Applikationen zufrieden bist - gerade in der jetzigen Situation - dann denke bitte darüber nach, dich bei diesen Projekten zu bedanken oder helfend einzubringen!
+
+Ließ diesen Newsletter über unseren RSS-Feed:
+- [Das englische Original](https://xmpp.org/feeds/all.atom.xml)!
+- [Die deutsche Übersetzung bei Anoxinon e.V.](https://anoxinon.de/tags/xmpp-newsletter/index.xml)
+
+Bist du daran interessiert, das Newsletter-Team zu unterstützen? Ließ mehr am Ende des Artikels.
+
+Abseits davon - viel Spaß beim Lesen!
+
+### Übersetzungen des Newsletters
+
+Die Übersetzungen des XMPP Newsletters werden auf den folgenden Seiten veröffentlicht (mit etwas Verzögerung):
+
+- Die französische Übersetzung erscheint auf [jabberfr.org](https://news.jabberfr.org/category/newsletter/) und [linuxfr.org](https://linuxfr.org/tags/xmpp/public)
+- Die deutsche Übersetzung erscheint auf [anoxinon.de](https://anoxinon.de/blog/)
+- Die italienische Übersetzung erscheint auf [NicFab.it](https://www.nicfab.it/)
+- Die spanische Übersetzung erscheint äquivalent zum englischen Original, wähle einfach [Spanisch in der oberen Leiste des Originalartikels](https://xmpp.org/2022/12/ES/newsletter-dec21-jan22/)
+
+Vielen Dank an die Übersetzenden und deren Arbeit! Das leistet eine klasse Hilfe dabei, die Neuigkeiten verfügbar zu machen! Bitte schließe dich deren Arbeit an oder starte neu mit einer weiteren Sprache!
+
+## XSF Ankündigungen
+
+- Die XSF plant, am [Google Summer of Code 2022 (GSoC) [EN]](https://xmpp.org/community/gsoc-2022/) teilzunehmen. Wenn du ein einer Teilnahme als Student:in interessiert bist oder auch als Mentor:in oder eigenständiges Projekt, dann [ergänze deine Ideen und nimmt mit uns Kontakt auf [EN]](https://wiki.xmpp.org/web/Google_Summer_of_Code_2022)!
```

Comment:
Can you take out the [EN] outside behind th link? And also everywhere else?

Reply:
Then there is no hyperlink behind it, but for sure, I will do.

Reply:
done